requirements/default.in

```diff
@@ -18,7 +18,7 @@ html5lib==1.0.1 	# MIT
 isoweek==1.3.3		# BSD
 numpy==1.16.5 		# BSD
 paypalrestsdk==1.9.0    # Paypal SDK License
-psycopg2==2.6.2     # LGPL
+psycopg2     # LGPL
```

Comment:
we should not remove the version if it is pinned

Reply:
I forgot to revert this change, updating